chapter1/OSGPProperties.md

```diff
@@ -0,0 +1,31 @@
+## I.1 Properties of the Open Smart Grid Platform
+
+- Acts as a **connecting link** between (web)applications and smart devices
+- The open Source approach **prevents vendor lock-in** 
+- State of the art **security** 
```

Comment:
This is only true for the OSLP devices. Other devices have less or different security. (The IEC61850 devices have no security at all). You may want to change this statement.